.github/workflows/release.yml

```diff
@@ -37,14 +49,40 @@ jobs:
         script: |
           try {
             await github.rest.repos.createRelease({
-              draft: false,
+              draft: process.env.DRAFT,
               generate_release_notes: true,
               name: process.env.RELEASE_TAG,
               owner: context.repo.owner,
-              prerelease: false,
+              prerelease: process.env.PRE_RELEASE,
               repo: context.repo.repo,
               tag_name: process.env.VERSION,
             });
           } catch (error) {
             core.setFailed(error.message);
-          }
\ No newline at end of file
+          }
+  Build:
+    name: Build Release with JNI
+    uses: ./.github/workflows/nix_release_builds.yml
+    secrets: inherit # pass all secrets
+    needs: [ Release ]
+    with:
+      release: true
+      version: ${{ github.event.inputs.version }}
+  Publish:
+    name: Publish to WD
+    needs: [Build]
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
```

Comment:
line duplication?

Reply:
Nope, first one clones libcore, the second clones wallet-daemon, I need both here

Reply:
I'll add names to make it more clear

Reply:
thks !